### PR TITLE
Fix editing text inside button

### PR DESCRIPTION
**Pull request by TrySound**
Ref https://github.com/webstudio-is/webstudio-builder/issues/1719

Buttons has issue with unavoidable space which prevents entering spaces. Here bind lexical to temporary span inside of button. Now space works fine.

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio-builder/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env.example` and the `builder/env-check.js` if mandatory

### CODE REVIEW

**Review by kof — APPROVED** [review]
what should we do about sites already using innerText?

kinda need a migration or keep supporting innerText

File: packages/prisma-client/prisma/migrations/20230610111903_button_children/migration.ts [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+            const instancesList: InstancesList = JSON.parse(build.instances);
+            const propsList: PropsList = JSON.parse(build.props);
+            const newPropsList: PropsList = [];
+            const buttonInstanceIds = new Set<string>();
+            const buttonInnerTexts = new Map<string, string>();
+
+            for (const instance of instancesList) {
+              if (
+                instance.component === "Button" ||
+                instance.component === "VimeoPlayButton"
```
Comment (kof) [inline packages/prisma-client/prisma/migrations/20230610111903_button_children/migration.ts:84]: btw that brings me to the question: how would we migrate someone else's components and my guess is we can't, if they upgrade component they have to redo them or the component author has to create a migration?
wdyt?

Comment (TrySound) [reply packages/prisma-client/prisma/migrations/20230610111903_button_children/migration.ts:84]: I guess similar to what we planned with sdk. Library migrations.

Comment (kof) [reply packages/prisma-client/prisma/migrations/20230610111903_button_children/migration.ts:84]: true, forgot about it

**Pull request merged**